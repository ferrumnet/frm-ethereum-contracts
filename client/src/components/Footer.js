import React from 'react';


const Footer = () => (
<div className="card text-white text-center bg-dark remove-borders" style={{maxWidth: "18rem;"}}>
  <div className="card-body">
    <h5 className="card-title">Ferrum Contracts</h5>
  </div>
  <div className="card-footer">copyright @2019 </div>
  </div>
);


export default Footer;
