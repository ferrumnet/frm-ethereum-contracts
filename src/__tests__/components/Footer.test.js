import Footer from "../../components/Footer";
import { shallow } from "enzyme";
import React from "react";

describe("Test Footer Component", () => {
  test("It should match the snapshot", () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
