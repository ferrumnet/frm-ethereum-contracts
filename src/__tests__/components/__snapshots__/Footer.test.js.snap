// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer Component It should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="card-body"
        >
          <h5
            className="card-title"
          >
            Ferrum Contracts
          </h5>
        </div>,
        <div
          className="card-footer"
        >
          copyright @2019 
        </div>,
      ],
      "className": "card text-white text-center bg-dark remove-borders footer",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h5
            className="card-title"
          >
            Ferrum Contracts
          </h5>,
          "className": "card-body",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Ferrum Contracts",
            "className": "card-title",
          },
          "ref": null,
          "rendered": "Ferrum Contracts",
          "type": "h5",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "copyright @2019 ",
          "className": "card-footer",
        },
        "ref": null,
        "rendered": "copyright @2019 ",
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="card-body"
          >
            <h5
              className="card-title"
            >
              Ferrum Contracts
            </h5>
          </div>,
          <div
            className="card-footer"
          >
            copyright @2019 
          </div>,
        ],
        "className": "card text-white text-center bg-dark remove-borders footer",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h5
              className="card-title"
            >
              Ferrum Contracts
            </h5>,
            "className": "card-body",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Ferrum Contracts",
              "className": "card-title",
            },
            "ref": null,
            "rendered": "Ferrum Contracts",
            "type": "h5",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "copyright @2019 ",
            "className": "card-footer",
          },
          "ref": null,
          "rendered": "copyright @2019 ",
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
